tests/integration/stepfunctions/v2/error_handling/utils.py

```diff
@@ -0,0 +1,41 @@
+from localstack.testing.snapshots.transformer import RegexTransformer
+from localstack.utils.strings import short_uid
+from tests.integration.stepfunctions.utils import await_execution_success
+
+
+@staticmethod
+def _test_sfn_scenario(
```

Comment:
```suggestion
def base_test_sfn_scenario(
```
nit: staticmethod isn't needed here and `_` should be reserved to denote "private" access in a module or class.